persistence-modules/hibernate-queries/src/test/java/com/baeldung/hibernate/keywords/HibernateKeywordsApplicationLiveTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.hibernate.keywords;
+
+import static java.util.UUID.randomUUID;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import jakarta.persistence.PersistenceException;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class HibernateKeywordsApplicationLiveTest {
```

Comment:
please let me know the article with which this changes is related?

Reply:
I just rename the name of this class because I saw it is trying to connect a real database. 
It is related to this article: https://www.baeldung.com/java-hibernate-db-keywords-as-columns

Reply:
I removed duplicated Unit test since there was already the HibernateKeywordsApplicationManualTest.